src/ripple/app/main/DBInit.h

```diff
@@ -116,9 +116,30 @@ std::array<char const*, 8> TxDBInit {{
 
 ////////////////////////////////////////////////////////////////////////////////
 
+// Temporary database used with an incomplete shard that is being acquired
+static constexpr auto AcquireDBName {"acquire.db"};
+
+static constexpr
+std::array<char const*, 3> AcquireDBPragma {{
+    "PRAGMA synchronous=NORMAL;",
+    "PRAGMA journal_mode=WAL;",
+    "PRAGMA journal_size_limit=1582080;"
+}};
+
+static constexpr
+std::array<char const*, 1> AcquireDBInit {{
+    "CREATE TABLE IF NOT EXISTS Shard (             \
+        ShardIndex          INTEGER PRIMARY KEY,    \
+        LastLedgerHash      CHARACTER(64),          \
+        StoredLedgerSeqs    BLOB                    \
+    );"
+}};
+
+////////////////////////////////////////////////////////////////////////////////
+
 // Pragma for Ledger and Transaction databases with complete shards
 static constexpr
-std::array<char const*, 2> CompleteShardDBPragma {{
+std::array<char const*, 2> CompltDBPragma {{
```

Comment:
Nit: Is this a typo or are you trying to shorten then variable name?

Reply:
Trying to shorten.